File: csharp/sprint5/A/Solution.cs
using System;

#if REMOTE_JUDGE
#else
public class Node
{
    public int Value { get; set; }
    public Node Left { get; set; }
    public Node Right { get; set; }

    public Node(int value)
    {
        Value = value;
        Left = null;
        Right = null;
    }
}
#endif

public class Solution
{
    public static int Solve(Node root)
    {
        // Your code
        // “ヽ(´▽｀)ノ”
    }


    private static void Test()
    {
        var node1 = new Node(1);
        var node2 = new Node(-5);
        var node3 = new Node(3)
        {
            Left = node1,
            Right = node2
        };
        var node4 = new Node(2)
        {
            Left = node3
        };
        Console.WriteLine(Solve(node5));
    }
}

File: swift/sprint5/A/solution.swift
#if !REMOTE_JUDGE
open class Node {

    public var value: Int
    public var left: Node?
    public var right: Node?

    public init(_ value: Int) {
        self.value = value
        self.right = nil
        self.left = nil
    }
}
#endif

func solution(node: Node?) -> Int {
    // Your code
    // ヽ(´▽`)/
    return 0
}

func test() {
    var node1 = Node(1)
    var node2 = Node(-5)
    var node3 = Node(3)
    node3.left = node1
    node3.right = node2
    var node4 = Node(2)
    node4.left = node3
    assert(solution(node: node4) == 3)
}
